src/coreclr/jit/emitxarch.cpp

```diff
@@ -991,6 +992,11 @@ emitter::code_t emitter::AddEvexPrefix(instruction ins, code_t code, emitAttr at
         // Set L bit to 1 in case of instructions that operate on 256-bits.
         code |= LBIT_IN_BYTE_EVEX_PREFIX;
     }
+    else if (attr == EA_64BYTE)
```

Comment:
Looks like there is a `TODO-Xarch-AVX512` just above that can be removed.

Reply:
Done